ui/component/or-attribute-card/src/index.ts

```diff
@@ -454,7 +454,14 @@ export class OrAttributeCard extends LitElement {
                                                     return option;
                                                 }),
                                                 this.period,
-                                                (value) => this._setPeriodOption(value)
+                                                (value) => this._setPeriodOption(value),
+                                                undefined,      // closedCallback
+                                                false,          // multiSelect
+                                                true,           // translateValues
+                                                false,          // midHeight
+                                                false,          // fullWidth
+                                                "kpi-menu-id",  // Menu Id
```

Comment:
Is there a need for a custom ID here? (since you're already using `ref()` for the element reference)

Reply:
Hi @MartinaeyNL , I used a custom ID here to avoid interfering with any existing behavior and to keep the changes scoped specifically to the KPI menu. This approach also gives us more flexibility to evolve each menu type independently in the future.

Reply:
Yeah indeed. I agree on the flexibility value of modifying the element ID here.
For example using `querySelector()` in a consumer element.
Might not be necessary with these changes, but it doesn't hurt adding it. 👍